src/components/CodeBlock/index.js

```diff
@@ -54,7 +57,7 @@ const Code = ({ codeString, language }) => {
       {...defaultProps}
       code={codeString}
       language={language}
-      theme={theme}
+      theme={isDark ? theme : vsCodeLight}
```

Comment:
keep the theme as it was earlier please. Currently light doesn't seems to good so we can keep the existing effect